src/components/SearchComponent.vue

```diff
@@ -58,18 +65,18 @@ const dropdownLabel = (text: string) => {
   />
 
   <div v-else>
-    <label id="search-label" for="select-search" class="fr-sr-only"
-      >Rechercher. Saisissez un mot clé puis choisissez une des options situés
+    <label :id="id + '-label'" :for="id" class="fr-sr-only">
+      Rechercher. Saisissez un mot clé puis choisissez une des options situés
       après le champ pour lancer la recherche dans la rubrique souhaitée</label
     >
     <Multiselect
-      id="select-search"
+      :id="id"
       v-model="selectedMultiSearch"
       class="select-search"
       :options="dropdown"
       label="text"
       track-by="route"
-      placeholder=""
+      :placeholder="placeholder"
       select-label=""
       deselect-label=""
       aria-labelledby="search-label"
```

Comment:
Il faut correspondre à l'`id` du `label`

```suggestion
      :aria-labelledby="id + '-label'"
```